src/Attachment.tsx

```diff
@@ -170,34 +169,44 @@ export const AttachmentView = (props: {
         case "application/vnd.microsoft.card.hero":
             if (!attachment.content)
                 return null;
+
+            const heroCardBuilder = new CardBuilder.AdaptiveCardBuilder();
```

Comment:
I actually had this in a past revision. Made it less maintainable because we have this big switch statement here which consolidates all the types, and didn't want to have a structure like that in 2 places.